modules/nf-core/untar/main.nf

```diff
@@ -0,0 +1,63 @@
+process UNTAR {
```

Comment:
You could update this to the very latest version of the UNTAR module in the nf-core repo, as this has integrated tests.